### PR TITLE
Documentation / Review readme

### PR DESCRIPTION
**Pull request by fxprunayre**

### CODE REVIEW

**Review by joachimnielandt — COMMENTED** [review]
There's a question remaining on line 193:

```
Q:
* check default view using `default='true'`. What happens if more than one default?
* check which view to load by default? Maybe see EditorController.js
```

Another one on line 303:

```
Q: Check how to enable/disable schematron rules for a profile with an example
```

I remember Olivia having an example here - I'll try to fish it up and perhaps also include it in the VL fork for reference.

File: README.md [hunk 1]
```diff
@@ -85,19 +89,19 @@ Add the module to `process-resources` phase of `web/pom.xml` to make sure it is
 
 Commit these changes.
 
-Apply the [patches](/core-geonetwork-patches) to the GeoNetwork core. You may need to manually apply specific hunks of a patch.
+Apply the [patches](core-geonetwork-patches) to the GeoNetwork core. You may need to manually apply specific hunks of a patch.
 
 ```
 # execute the following in the core-geonetwork root directory 
-git am --ignore-space-change --ignore-whitespace --reject --whitespace=fix schemas/dcat-ap/core-geonetwork-patches/*.patch
+git am --ignore-space-change --ignore-whitespace --reject --whitespace=fix core-geonetwork-patches/*.patch
```
[the remark this answers is not in the document]
Comment (fxprunayre) [reply README.md:96]: Indeed. I'll fix this

**Comment by fxprunayre** [comment]
> * check default view using `default='true'`. What happens if more than one default?

It fails. Improved in https://github.com/geonetwork/core-geonetwork/pull/8548

> * check which view to load by default? Maybe see EditorController.js

With the PR above, we can put the first default view (with or without condition) to use first and it should work.

The order in the menu is based on the order in config-editor
image

Maybe we can remove `fromRoot` which was more for dev purpose and we can also hide `xml` view for non admin ?

**Comment by joachimnielandt** [comment]
Hiding `fromRoot` seems reasonable, but I think a lot of our editors are still accessing the XML view from time to time. I would keep that available, for sure.

**Pull request merged**